test/controllers/survey_controller_test.exs

```diff
@@ -130,7 +132,7 @@ defmodule Ask.SurveyControllerTest do
   describe "update" do
     test "updates and renders chosen resource when data is valid", %{conn: conn, user: user} do
       project = insert(:project, user: user)
-      survey = insert(:survey, project: project, schedule_start_time: Ecto.Time.cast!("09:00:00"), schedule_end_time: Ecto.Time.cast!("18:00:00"))
+      survey = insert(:survey, project: project, schedule_start_time: Ecto.Time.cast!("09:00:00"), schedule_end_time: Ecto.Time.cast!("18:00:00"), timezone: "Etc/UTC")
```

Comment:
Set default `timezone` in the `survey_factory` function (factory.ex) instead of passing the value everywhere?